Update old blog article about SIG Usability to mention dissolution (#42390)
kubernetes/website: content/en/blog/_posts/2021-07-15-SIG-Usability-Spotlight.md

slug: sig-usability-spotlight-2021
---

**Author:** Kunal Kushwaha (Civo)

{{< note >}}

> SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.

Reviewer: ```suggestion
SIG Usability, which is featured in this Spotlight blog, has been retired; it is no longer active.
```

Author: I believe we can go with 'deprecated and'

Reviewer: OK. “Deprecated” is **definitely** wrong, but overall this PR is still clearly helping readers.

**Author:** Kunal Kushwaha, Civo

_**Note: SIG Usability is Not active.**_
_SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.

> As a result, the links and information provided in this post may no longer be valid or relevant.

Reviewer 2: ```suggestion
As a result, the links and information provided in this blog post may no longer be valid or relevant.
```

> Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the k8s/community policy.

Reviewer 2: ```suggestion
Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
```

> We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._

Reviewer 2: ```suggestion
We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._
```
Instead of pointing out directly, maybe we can provide a reference link.

Author: what reference link? aren't they already on the kubernetes website if reading the blog🤔
or https://kubernetes.io/community/

Reviewer 2: In other words we can refer them to _k/community_ repo by adding a reference ink at the last of the text you've writtten.

> _**Note: SIG Usability is Not active.**_
> _SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
> As a result, the links and information provided in this blog post may no longer be valid or relevant.
> Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
> We encourage you to explore other SIGs and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes usability._
> 

Reviewer 3: It would be beneficial to utilize the `{{< note >}}` shortcode for incorporating this content. 

```suggestion
{{< note >}}
SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
As a result, the links and information provided in this blog post may no longer be valid or relevant.
Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.
We encourage you to explore other [SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups) and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes.
{{< /note >}}

```

Reviewer: I avoid using `{{< note >}}` shortcodes in blog articles because they may one day end up as more difficult to maintain.

It's not a hard restriction, just something to bear in mind.

> **Author:** Kunal Kushwaha, Civo

Reviewer: (nit)
```suggestion
**Author:** Kunal Kushwaha (Civo)
```

{{< note >}}
SIG Usability, which is featured in this Spotlight blog, has been deprecated and is no longer active.
As a result, the links and information provided in this blog post may no longer be valid or relevant.

> Should there be renewed interest and increased participation in the future, the SIG may be revived. However, for the time being, it will be inactive as per the Kubernetes community policy.

Reviewer: (wrapping)
```suggestion
Should there be renewed interest and increased participation in the future, the SIG may be revived.
However, as of August 2023 the SIG is inactive per the Kubernetes community policy.
```

> We encourage you to explore other [SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups) and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes.

Reviewer: There's one author (Kunal), so who does “we” refer to here?

Author: In this context, "we" refers to the Kubernetes community and not the author of the blog post. It's common for community announcements, updates to use "we" to represent the collective voice of the organization providing the information. In this case, "we" includes the community folks who are informing readers about the deprecation of SIG Usability and encouraging them to explore other resources and SIGs within the Kubernetes community.

Author: in the context of Note

Reviewer: > In this context, "we" refers to the Kubernetes community and not the author of the blog post. It's common for community announcements, updates to use "we" to represent the collective voice of the organization providing the information. In this case, "we" includes the community folks who are informing readers about the deprecation of SIG Usability and encouraging them to explore other resources and SIGs within the Kubernetes community.

That's fine, but: don't explain this only to the reviewers. Please make it clear to readers too.

> We encourage you to explore other [SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups) and resources available on the Kubernetes website to stay up-to-date with the latest developments and enhancements in Kubernetes.

Reviewer: ```suggestion
The Kubernetes project encourages you to explore other
[SIGs](https://github.com/kubernetes/community/blob/master/sig-list.md#special-interest-groups)
and resources available on the Kubernetes website to stay up-to-date with the latest developments
and enhancements in Kubernetes.
```
